include/silo/storage/aa_store.h

```diff
@@ -44,7 +44,7 @@ class AAPosition {
    explicit AAPosition(AA_SYMBOL symbol);
    explicit AAPosition(std::optional<AA_SYMBOL> symbol);
 
-   AASymbolMap<roaring::Roaring> bitmaps;
+   SymbolMap<AA_SYMBOL, roaring::Roaring> bitmaps;
    std::optional<AA_SYMBOL> symbol_whose_bitmap_is_flipped = std::nullopt;
 
    std::optional<silo::AA_SYMBOL> flipMostNumerousBitmap(uint32_t sequence_count);
```

Comment:
Not shown here, but `computeSize`, `runOptimize` and `shrinkToFit` are not implemented.